pages/access/[serviceId]/[environmentId]/[resourceId]/[resourceInstanceId].js

```
@@ -513,13 +513,10 @@ function getTabs(
   if (isMetricsEnabled && !isResourceBYOA && !isCliManagedResource)
     tabs["metrics"] = "Metrics";
-  if (isLogsEnabled && !isResourceBYOA && !isCliManagedResource)
-    tabs["logs"] = "Logs";
+  if (isLogsEnabled && !isResourceBYOA) tabs["logs"] = "Logs";
 
-  if (!isActive || isCliManagedResource) {
-    if (!isActive || resourceType === RESOURCE_TYPES.Terraform) {
-      delete tabs.connectivity;
-    }
+  if (!isActive || resourceType === RESOURCE_TYPES.Terraform) {
+    delete tabs.connectivity;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)